vpc-endpoints.tf

```diff
@@ -0,0 +1,35 @@
+resource "aws_security_group" "sgforendpoint" {
+  name        = "EndpointSG"
+  description = "Allow indbound and outbound traffic for VPC endpoint"
+  vpc_id      = "${aws_vpc.main.id}"
+
+  ingress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    # Please restrict your ingress to only necessary IPs and ports.
+    # Opening to 0.0.0.0/0 can lead to security vulnerabilities.
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
Perhaps we should make this configurable? Something like 
```
[
    {
        vpc_endpoint = com.amazonaws.x.y
        cidr_blocks = [] (Optional)
        source_security_group_id = "" (Optional) 
    }
]
```

Reply:
I have added https://github.com/cmdlabs/terraform-aws-vpc/issues/7#issue-492551477 to address this enhancement